services/movie_session.py

```diff
@@ -42,3 +42,11 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
```

Comment:
The return type annotation for `get_taken_seats` should specify `list[dict[str, int]]` to indicate that the list contains dictionaries with string keys and integer values.